gulpfile.babel.js

```diff
@@ -66,5 +66,5 @@ gulp.task('default', (done) => {
   if (args.production)
     runSequence('server', done);
   else
-    runSequence('server', 'karma-dev', done);
+    runSequence('server', done);
```

Comment:
Hmm, we should run Karma only for DOM involved tests, and run our unit tests for TDD in Node.js only.